doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+Follow these commands to generate the ``acrn.efi`` with industry scenario::
+
+  $ cd ~
+  $ export https_proxy=<your proxy>:<proxy port>
+  $ git clone http://github.com/projectacrn/acrn-hypervisor.git
+  $ cd acrn-hypervisor
+  $ git checkout acrn-2019w31.5-140000p
+  $ sudo vim hypervisor/arch/x86/Kconfig
+  # <Change the ACRN scenario to INDUSTRY and save>
+  choice
+         prompt "ACRN Scenario"
+         default INDUSTRY
+  $ make BOARD=kbl-nuc-i7 FIRMWARE=uefi
+  $ ls build/hypervisor/acrn.efi                    #  You will get hypervisor in this directory.
+
+Replace the acrn.efi to support industry on Service VM (root privilege)
+=======================================================================
+Now we just need to follow the steps to enable the industry scenario for Service VM::
+
+  # mount /dev/sda1 /mnt
+  # mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.bak
+  # scp <build machine>:~/acrn-hypervisor/build/hypervisor/acrn.efi /mnt/EFI/acrn/
+  # sync && umount /mnt
+  # reboot
+
+How to rebuild Service VM kernel from acrn-kernel tree
+******************************************************
+In this section, we will learn to build Service VM kernel and then update it onto KBL NUC.
+
+Build the Service VM kernel on build machine (non-root privilege)
+=================================================================
+Follow these commands to generate the Service VM kernel on your build machine::
+
+  $ cd ~
+  $ export https_proxy=<your proxy>:<proxy port>
+  $ git clone http://github.com/projectacrn/acrn-kernel.git
+  $ cd acrn-kernel
+  $ git checkout acrn-2019w31.5-140000p
+  $ cp kernel_config_sos .config
+  $ make menuconfig   ## Don't do anything, just save and quit to set the default config items
+  $ make bzImage && make modules
+  $ make modules_install INSTALL_MOD_PATH=$PWD
+
+Update the Service VM kernel
+============================
+Follow the steps to enable the new Service VM kernel and its modules::
+
+  # mount /dev/sda1 /mnt
+  # scp <build machine>:~/acrn-kernel/arch/x86/boot/bzImage /mnt/EFI/org.clearlinux/
+  # sed -i "s/$(sed -n '2p' /mnt/loader/entries/acrn.conf | cut -d'/' -f4)/bzImage/" /mnt/loader/entries/acrn.conf
+  # scp -r <build machine>:~/acrn-kernel/lib/modules/* /usr/lib/modules/
+  # sync && umount /mnt
+  # reboot
+
+How to update Preempt-RT kernel
+*******************************
+In this section, we will learn to update the Preempt-RT kernel by two methods:
+
+- Rebuild from acrn-kernel branch tree.
+- Install Preempt-RT kernel from Clear Linux bundles.
+
+You could try either of them to get the Preempt-RT kernel and then copy it onto NVMe disk.
+
+How to rebuild the Preempt-RT kernel from acrn-kernel branch tree
+=================================================================
+#. Build the Preempt-RT kernel on your build machine (non-root privilege)::
+
+   $ cd ~/acrn-kernel
+   $ git reset --hard
+   $ git clean -df
+   $ git checkout acrn-2019w31.5-143000p
```

Comment:
should this be an official released version tag (e.g., v1.2?)

Reply:
There is no v1.2 tag in acrn-kernel repo